src/com/comeet/utilities/Validator.java

```diff
@@ -0,0 +1,15 @@
+package com.comeet.utilities;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator {
+    
+    private static final Pattern VALID_EMAIL_ADDRESS_REGEX = 
```

Comment:
now, *exchange* addresses might be more restrictive. but I'm not going to go digging up documentation on this.